bundle/manifests/kuadrant.io_kuadrants.yaml

```diff
@@ -63,7 +63,6 @@ spec:
                     message:
                       description: message is a human readable message indicating
                         details about the transition. This may be an empty string.
```

Comment:
probably some generated reason, but wondering why this definiton changed?

Reply:
Not sure either 🤔

If I run `make generate manifests bundle` in `main`, there are no changes. On this branch on the other hand, it gives me this.

```
❯ go version
go version go1.21.2 darwin/arm64

❯ ./bin/controller-gen --version
Version: v0.10.0

❯ ./bin/operator-sdk version
operator-sdk version: "v1.28.1", commit: "b05f6a56a176a98b7d92c4d4b36076967e0d77f7", kubernetes version: "1.26.0", go version: "go1.19.8", GOOS: "darwin", GOARCH: "arm64"

❯ ./bin/opm version
Version: version.Version{OpmVersion:"v1.26.2", GitCommit:"0e53cafe", BuildDate:"2022-09-21T15:25:25Z", GoOs:"darwin", GoArch:"arm64"}

❯ ./bin/kustomize version
{Version:unknown GitCommit:$Format:%H$ BuildDate:1970-01-01T00:00:00Z GoOs:darwin GoArch:arm64}
```